### ISSUE TITLE
On launch of this project into VSCode, it shows 129 problems

### ISSUE THREAD

**Issue by ghedwards** [SimpleBoilerplates/Flutter#23]
On launch of this project into VSCode, it shows 129 problems, I believe most of these to be information or warnings, but was wondering it was worth trying to fix most of them ?

**Comment by Sadmansamee** [comment]
Yes, most of them are information or warnings, and definitely worth trying to fix them, I was busy with work so couldn't try. are you interested to fix them? you are welcome if you are.

**Comment by xyl-san** [comment]
#28 Fixed some of the 150+ lint problems.
